redfish-core/lib/oem/ibm/lamp_test.hpp

```diff
@@ -28,12 +28,19 @@ inline void getLampTestState(const std::shared_ptr<bmcweb::AsyncResp>& aResp)
                 getObjectType) {
             if (ec)
             {
+	        if(ec == boost::system::errc::host_unreachable)
+		{
+		    BMCWEB_LOG_DEBUG << "lamp test not available yet";
+		    std::cout << " lamp test not available";

```

Comment:
we don't use cout

Reply:
Yeah, sorry this patch came up unintentionally.